package.json

```diff
@@ -61,7 +61,7 @@
     "prepare": "is-ci || husky install",
     "prepublishOnly": "npm run build",
     "postpublish": "npm run clean",
-    "test": "jest",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
Necessary for esm

Reply:
Alternatively we could [ask contributors](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#testing-package-locally) to use at least v14, but this is fine as well. Though I'd say it is ideally removed when LTS for v12 ends ([on 2022-04-30](https://nodejs.org/en/about/releases/))

Reply:
This is necessary even if you're on the latest Node version ([jest docs](https://jestjs.io/docs/ecmascript-modules)).

Reply:
Thanks for pointing that out, didn't realize it's a Jest requirement :slightly_smiling_face: